boards/FSM/Src/App/App_FlowMeter.c

```diff
@@ -1,12 +1,12 @@
-#include <stddef.h>
 #include <assert.h>
-#include "App_FlowMeter.h"
+#include "App_CanTx.h"
```

Comment:
This must be a typo? CanTx doesnt' belong here and isn't used anyway
```suggestion
#include "App_FlowMeter.h"
```